.github/workflows/dependabot-batcher.yml

```diff
@@ -0,0 +1,38 @@
+# Dependabot Batcher v1.0.1
+
+name: 'Dependabot Batcher'
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "r/12.x", "develop" ]
+  pull_request:
+    branches: [ "r/12.x", "develop" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  dependabot-batcher:
+    name: 'Combine dependabot PRs'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repo'
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPENDABOT_BATCHER }}
+
+      - name: 'Set Node Version'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.8.0
```

Comment:
Does `dependabot-batche` need this?

Reply:
To be honest, I'm not very sure, I based my code in the implementation from the creators of this action https://github.com/Legal-and-General/dependabot-batcher/blob/main/.github/workflows/dependabot.scheduled.yml

Reply:
You were right!, it's not necessary for the action to work.